.github/workflows/build.yml

```
@@ -14,7 +14,7 @@ jobs:
             fail-fast: false
             matrix:
                 os: [ubuntu-latest]
-                php: ['5.5', '5.6', '7.0', '7.1', '7.2', '7.3', '7.4', '8.0', '8.1']
+                php: ['7.0', '7.1', '7.2', '7.3', '7.4', '8.0', '8.1']
```

Maybe we can add 8.2 also.
```suggestion
                php: ['7.0', '7.1', '7.2', '7.3', '7.4', '8.0', '8.1', '8.2']
```